### DISCUSSION TITLE
Wish "display mode" can be added

### DISCUSSION THREAD

**Discussion by tisonwu** [seerge/g-helper#1368] [Ideas]
My laptop is 2023 Z16, and I wish that the feature for switching display mode of the screen can be added. That will be very helpful when I play games or watch movies

**Comment by seerge** [comment]
@tisonwu i have answered on the [duplicate issue page](https://github.com/seerge/g-helper/issues/1369) you created

**Comment by seerge** [comment]
Added in latest release https://github.com/seerge/g-helper/releases
